jqjq.jq

```diff
@@ -1118,6 +1117,81 @@ def parse:
   // error("parse error: \(.)")
   );
 
+def _tojson($opts):
+  # color order: null, false, true, number, string, array, object, field
+  def _color($id):
+    if $opts.colors != null then "\u001b[\($opts.colors[$id])m"
+    else empty end;
+  def _reset_color:
+    if $opts.colors != null then "\u001b[0m"
+    else empty end;
+  def _wrap_color($id):
+    if $opts.colors != null then _color($id) + . + _reset_color
+    else . end;
+  def _f($opts; $indent):
+    def _r($prefix):
+      ( type as $t
+      | if $t == "null" then tojson | _wrap_color(0)
```

Comment:
Since they're local and not useful elsewhere (in the jq source, they're only used in `jv_dump_term`), I named them shorter like `_c_null`. They're only used in `_wrap_color` and `_color` calls, so their usage makes it clear they're color IDs.